js/src/components/contact-information/phone-number-card/verification-code-control.scss

```diff
@@ -1,7 +1,7 @@
 .gla-verification-code-control {
 	// Verification code input
 	.components-input-control .components-input-control__container .components-input-control__input {
-		width: 40px;
+		width: $grid-unit-50;
```

Comment:
❔ Looks like the grid unit is more intently to be used in spacing. Would here be better to use a dimension value?

Reply:
I see all over the codebase of WP / WC using grid units for sizes. I see them as a grid relative unit size, so makes sense for me to use them also for width, height etc...